mjolnir_testing.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# This script only exists so that the persitent state in volumes
```

Comment:
I think it would be worth turning this immediately into a Node script.

It might be able to replace the registration Dockerfile, too.